packages/discovery-provider/src/api/v1/models/tracks.py

```diff
@@ -72,7 +72,13 @@
 track = ns.model(
     "Track",
     {
+        "access": fields.Nested(
+            access, description="Describes what access the given user has"
+        ),
         "artwork": fields.Nested(track_artwork, allow_null=True),
+        "blocknumber": fields.Integer(
```

Comment:
This field means a bunch of different things in different contexts, but I guess this description is probably sufficient.

Reply:
to be clear this is the blocknumber _on the track_ not the one at the root of the response - I think this is the correct description?

Reply:
yeah that's why it could be confusing. in a perfect world maybe updated_at_block_number